GCTC-NTGC/gc-digital-talent
# Debt - 3899 - Remove Pool Placeholders

Resolves #3899 

## Summary

This removes some unused placeholder pages and routes we had for direct intake that were never used.

## Testing

1. Build talent search `npm run production --workspace=talentsearch`
2. Login and navigate to "Brows Opportunities" in the nav bar
3. Click on one of the pool and ensure you land on the advertisement page (not the placeholder)
4. Apply for a pool
5. Click the "Edit X" links in the profile section (step one)
6. Ensure the breadcrumbs work for the pool advertisement and steps
